src/main/java/org/terasology/MineSweeper/system/MinesweeperSystem.java

```diff
@@ -61,29 +61,54 @@
 
 /**
  * Created by michaelpollind on 8/28/16.
+ * System for handling minefields.
  */
 @RegisterSystem(RegisterMode.AUTHORITY)
 public class MinesweeperSystem extends BaseComponentSystem {
 
+    /**
```

Comment:
Remove javadocs on all of these fields.
- EntityManager
- Logger
- BlockEntityRegistry
- WorldProvider
- BlockManager